server/manual-rendering.md

```diff
@@ -0,0 +1,63 @@
+### Manual rendering
```

Comment:
Instead of showing all the compile execute functions in this section I suggest that we just call it handlebars-basics and step through each example with data, template and output. e.g.

*data*
```
var data = {para: 'I am a string'}
```

*template*
```
<div>{{para}}</div>
```

*output*
```
<div>I'm a string</div>
```